subworkflows/local/fasta_windows.nf

```diff
@@ -0,0 +1,77 @@
+//
+// Run fasta_windows and prepare all the output files
+//
+
+include { EXTRACT_COLUMN          } from '../../modules/local/extract_column'
+include { FASTAWINDOWS            } from '../../modules/nf-core/modules/fastawindows/main'
+include { TABIX_BGZIP             } from '../../modules/nf-core/modules/tabix/bgzip/main'
+include { TABIX_TABIX as TABIX_TABIX_CSI   } from '../../modules/nf-core/modules/tabix/tabix/main'
+include { TABIX_TABIX as TABIX_TABIX_TBI   } from '../../modules/nf-core/modules/tabix/tabix/main'
+
+// List of the columns we want to extract as bedGraph from the frequency files,
+// with the subdirectory name and the relevant part of the file name
+ch_freq_config = Channel.of(
+    [4,  'base_content/k1', 'GC'],
+    [5,  'base_content/k1', 'GC_skew'],
+    [6,  'base_content/k1', 'AT_skew'],
+    [7,  'base_content/k1', 'nucShannon'],
+    [8,  'base_content/k1', 'G'],
+    [9,  'base_content/k1', 'C'],
+    [10, 'base_content/k1', 'A'],
+    [11, 'base_content/k1', 'T'],
+    [12, 'base_content/k1', 'N'],
+    [13, 'base_content/k2', 'CpG'],
+    [14, 'base_content/k2', 'dinucShannon'],
+    [15, 'base_content/k3', 'trinucShannon'],
+    [16, 'base_content/k4', 'tetranucShannon'],
+)
```

Comment:
Is this fairly constant that you can hardcode it? Would it better to create using a fixed `assets` file?

Reply:
It's tied to that version of fasta_windows. Further versions may add more columns and shift things around, like between v0.2.3 and v0.2.4.
Happy to move the content to a csv file, and initialise the channel with `splitCsv` . I could even register the asset as a pipeline parameter in `nextflow.config`, allowing users to run the pipeline with a different csv file that would only have the columns they're interested in. In the same vein, I could also add flags to load/skip the mono/di/tri/tetra-nucleotide tsv files

Reply:
I've got a version of the pipeline that obeys this config file, which is also used to name the output of the mono/di/tri/tetra-nucleotide files. To skip an analysis, one can simply simply remove it from the csv. I'll push it later this morning